### PR TITLE
feat: remove client-side-routing code

### PR DESCRIPTION
**Pull request by chanceaclark**
## What

Removes client-side routing from the state as it's being tightly coupled to Next.js routing. We want to implement client side routing, but need to think of a better way to integrate it into state.

### CODE REVIEW

File: packages/runtime/src/index.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export {
   scrollDocumentElement,
   unregisterDocument,
   setBuilderEditMode,
-  changePathname,
```
Comment (migueloller) [inline packages/runtime/src/index.ts:15]: This function is being used in the builder. Is the intent to remove the call-site as well when we upgrade the builder to this version of the runtime?

File: packages/runtime/src/state/actions.ts [hunk 2]
```diff
@@ -72,10 +72,6 @@ export const ActionTypes = {
   MAKESWIFT_CONNECTION_INIT: 'MAKESWIFT_CONNECTION_INIT',
   MAKESWIFT_CONNECTION_CHECK: 'MAKESWIFT_CONNECTION_CHECK',
 
-  CHANGE_PATHNAME: 'CHANGE_PATHNAME',
-  CHANGE_PATHNAME_START: 'CHANGE_PATHNAME_START',
-  CHANGE_PATHNAME_COMPLETE: 'CHANGE_PATHNAME_COMPLETE',
```
Comment (migueloller) [inline packages/runtime/src/state/actions.ts:77]: Unfortunately, these are also used in the builder (the entire `ActionTypes` object is 😢), so is the plan to remove that code from the builder?

**Review by migueloller — APPROVED** [review]

File: packages/runtime/src/index.ts (hunk 1 above)
Comment (chanceaclark) [reply packages/runtime/src/index.ts:15]: Yeah, I have a branch ready to throw up in the builder to remove this.

Comment (migueloller) [reply packages/runtime/src/index.ts:15]: Ok, cool. I think we should ship that first, then.

Comment (chanceaclark) [reply packages/runtime/src/index.ts:15]: Submitted an MR for that.

**Review by migueloller — APPROVED** [review]

**Pull request merged**